.github/workflows/security.yaml

```diff
@@ -16,29 +16,6 @@ env:
   GO_VERSION: "1.19"
 
 jobs:
-  gosec:
-    runs-on: ubuntu-latest
-    steps:
-
-      - name: Checkout
-        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
-
-      - name: Run Gosec Security Scanner
-        uses: securego/gosec@1af1d5bb49259b62e45c505db397dd2ada5d74f8 # v2.14.0
-        with:
-          # Ignoring:
-          # G101: Potential hardcoded credentials
-          # G104: Errors not checked
-          # G304: Potential file inclusion via variable
-          # G306: Expect WriteFile permissions to be 0600 or less
-          # G307: Deferring unsafe method "Close"
-          args: -exclude=G101,G104,G304,G306,G307 -tests=false -exclude-dir=doc/ -no-fail -fmt sarif -out gosec-results.sarif  ./...
```

Comment:
If I'm not mistaken, removing the `-no-fail` option will stop failing and blocking the merge. Also, we might not need to remove this job with #459.